daemon/src/position_metrics.rs

```diff
@@ -271,6 +275,7 @@ impl sqlite_db::ClosedCfdAggregate for Cfd {
             position,
             quantity_usd,
             state,
+            counterparty_network_identity,
```

Comment:
🤡 I think this will need to be changed to the `PeerId` at some point. But we probably can't yet because not all positions have an associated `PeerId` (since it's an `Option` in the `Cfd`).

Reply:
Yes, once every peer has a peer_id we can change this. For now, every peer has a network identity .